src/content/docs/licenses/product-or-service-licenses/mobile-app-licenses/android-application-licenses.mdx

```diff
@@ -40,7 +40,7 @@ We love open-source software, and use the following in the New Relic Android app
       </td>
 
       <td>
-        Copyright 2011, Artem Votincev (apmem.org)
+        Copyright © 2011, Artem Votincev (apmem.org)
```

Comment:
I would remove Copyright, like in the instances above. Same goes for the rest of files.

Reply:
According to what Joseph told us (https://newrelic.atlassian.net/browse/DOC-6763), for third part libraries, we should keep `Copyright ©`.

Reply:
They're technically the same, but with respect to New Relic's use of third-party code, I would rather dot every "i" as stated in the license terms